source/manual/alerts/whitehall-scheduled-publishing.html.md.erb

```diff
@@ -42,3 +42,14 @@ recently restored, perhaps due to the data sync. In that case, you can try
 running the following Rake task on a `whitehall_backend` machine:
 
 <%= RunRakeTask.links("whitehall", "publishing:scheduled:requeue_all_jobs") %>
+
+Due to the overnight (data anonymisation process)[https://github.com/alphagov/whitehall/blob/7b5c5a086b89cb62ffba62b152a0a8dcfc10c8e6/script/scrub-database] you may notice
```

Comment:
```suggestion
Due to the overnight [data anonymisation process](https://github.com/alphagov/whitehall/blob/7b5c5a086b89cb62ffba62b152a0a8dcfc10c8e6/script/scrub-database) you may notice
```

Reply:
Thanks @issyl0 - I'm always getting those the wrong way.  Fixed.